src/reader-browser.js

```diff
@@ -139,6 +140,22 @@ export class CarReader {
     }
   }
 
+  /**
+   *
+   * @static
+   * @memberof CarReader
+   * @param {Uint8Array} bytes
+   * @returns {CarReader} blip blop
+   */
+  static fromBytesSync (bytes) {
+    if (!(bytes instanceof Uint8Array)) {
+      throw new TypeError('fromBytes() requires a Uint8Array')
+    }
+
+    const { header, blocks } = DecoderSync.fromBytes(bytes)
+    return new CarReader(header, blocks)
+  }
+
```

Comment:
I find it odd that it is browser specific functionality, I feel like it should be usable in node as well